### ISSUE TITLE
ssrnet_ori_mt result

### ISSUE THREAD

**Issue by hegc** [shamangary/FSA-Net#26]
Hi, can you share the model of ssrnet_ori_mt and the test MAE on AFLW2000?
My validation mae is about 10 when train mae is 3..

**Comment by RomanticWithoutStatus** [comment]
Oh my validation on AFLW2000 is also like you.  And when I add dropout layer，the same thing happens. don't know how the author can solve this situation.

**Comment by hegc** [comment]
@RomanticWithoutStatus How is about the model of ssrnet_mt which use separable convolution? I'm trying it in mxnet.

**Comment by RomanticWithoutStatus** [comment]
> @RomanticWithoutStatus How is about the model of ssrnet_mt which use separable convolution? I'm trying it in mxnet.

lol   I also trying in mxnet(also in caffe).  models only using separable convolution or only using convolution, I have try them,  and the same thing happens. my project(in chinese): https://blog.csdn.net/pc9803/article/details/102603689

**Comment by hegc** [comment]
@shamangary Hi, can you share your point?

**Comment by shamangary** [comment]
Hello @RomanticWithoutStatus and @hegc,
In the paper, I design a new two-stream backbone as described in our paper Sec. 3.6.

As for 'ssrnet_ori_mt', it is only my test code for reusing the original SSR-Net two-stream backbone.
The new backbone is used in 'ssrnet_mt' and it's the paper 'ssrnet-MD'.

As far as I know, the mxnet version of the SSR-Net was implemented based on the old backbone.
Please check the new one.

However, I found that my training code for 'ssrnet_mt' is not as same as I do before. Maybe its about the batchsize or other details.
Please consider to reduce the batchsize to get better results.

**Comment by hegc** [comment]
Thanks. I rewrite the 'ssrnet_mt' in mxnet. 
However, when I use Adam optimizer and batch_size 32, the val(BIWI) mae reach 6.4. 
Then I use batch_size 4, the val mae is about 9.3, the train mae is about 2.8. Maybe the model get overfitting.
I have another question, the stage num [5, 5, 5] would be better than [3, 3, 3] ? why do you choose [3, 3, 3]?

**Comment by shamangary** [comment]
Hello @hegc, the truth is I already delete my original setting so I don't really remember the batchsize when I test the ssrnet_mt. But the batchsize does affect a lot on the results. So the trivial way would be try batchsize [8,16,32,64] and pick the best one.

The reason I choose [3,3,3] is simply because it's the simplest one and the job is good enough. Adding more neurons will cost more memory and time.

**Comment by hegc** [comment]
Thank you,  the batch-size does affect a lot on the validation result.

**Comment by shamangary** [comment]
As a note, I use the new code and traing ssrnet_mt with batchsize=8
```
--------------------------------------------------------------------------------
ssrnet_mt, AFLW2000(300W_LP), MAE = 5.808, [yaw,pitch,roll] = [4.822, 7.219, 5.382]
--------------------------------------------------------------------------------

--------------------------------------------------------------------------------
ssrnet_mt, BIWI(300W_LP), MAE = 4.768, [yaw,pitch,roll] = [4.595, 6.316, 3.394]
--------------------------------------------------------------------------------

```

**Comment by hegc** [comment]
Yes, I can get same result as yours now in mxnet.
